src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/ContentResultExecutor.cs

```diff
@@ -4,11 +4,12 @@
 using System;
 using System.Text;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Internal;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.AspNetCore.Mvc.Internal
+namespace Microsoft.AspNetCore.Mvc.Infrastructure
```

Comment:
Is the namespace change to facilitate subclassing?

Reply:
Yes